### ISSUE TITLE
Peer2PeerTest fails

### ISSUE THREAD

**Issue by TheNetStriker** [jitsi/jitsi-meet-torture#267]
The Peer2PeerTest always fails on my Jitsi installation. I did take a look at the code and my guess ist that the method APP.conference.getP2PConnectionState() should be used to check the p2p connection instead of APP.conference.getConnectionState().

When I change the method the test is successful on my system.

Could you please check if this is an error in the code or am I doing something wrong here?

**Comment by damencho** [comment]
You are right, we haven't ran this test in a while, it was flaky at some point and we disabled it. Feel free to open a PR.

**Comment by TheNetStriker** [comment]
@damencho Thanks for the quick reply. I just created a PR: https://github.com/jitsi/jitsi-meet-torture/pull/268
